packages/dcos-integration-test/extra/test_mesos.py

```diff
@@ -9,6 +9,8 @@
 import test_helpers
 from dcos_test_utils import marathon, recordio
 
+__maintainer__ = 'adam-mesos, Gilbert88'
```

Comment:
Adam is leaving Mesosphere, right?

Reply:
Since this is an open source repo, I thought, it is okay to include him?

Reply:
@adam-mesos What do you think? It would be great to have you still around :)